python/glow/wgr/functions.py

```diff
@@ -158,8 +158,10 @@ def reshape_for_gwas(spark: SparkSession, label_df: pd.DataFrame) -> DataFrame:
         transposed_df = label_df.T
         column_names = ['label', 'values']
     elif label_df.index.nlevels == 2:  # Indexed by sample id and contig name
-        sample_ordering = pd.unique(label_df.index.get_level_values(0))
-        transposed_df = label_df.T.stack().reindex(sample_ordering, axis=1)
+        # stacking orders by the new index, so we need to remember the original sample order in case
```

Comment:
I'm a little confused by the grammar here - can you rephrase?

Reply:
Done